worker/src/RTC/Transport.cpp

```diff
@@ -106,7 +106,12 @@ namespace RTC
 
 		if (hasCallId) {
 			MS_DEBUG_TAG(rtp, "XXXXX creating consumer bin log. lively=%s", lively.ToStr().c_str());
-			this->consumersBinLog.InitLog('c', lively.callId, lively.id); // initialize consumers bin log here, it is shared btw all consumers
+
+            // initialize consumers bin log here, it is shared btw all consumers
+            std::string const callId = lively.callId;
+            this->consumersBinLog.InitLog([callId](uint64_t timestamp) -> std::string {
```

Comment:
why passing a function then storing the function instead of simply passing a string template ?